lerna.json

```diff
@@ -2,15 +2,6 @@
   "version": "independent",
   "lerna": "2.0.0",
   "npmClient": "yarn",
-  "commands": {
```

Comment:
All of these options are irrelevant when using workspaces, so just removed them to avoid confusion.